apps/PCLWrapper/pcl/registration/super4pcs.h

```diff
@@ -137,13 +135,28 @@ namespace pcl
       virtual ~Super4PCS ()
       {
       }
+      
+      /** \brief Setter for the delta (see paper).
+       */
+      inline void setDelta(float delta) { options_.delta = delta; }
+      inline void setSampleSize(int sampleSize) { options_.sample_size = sampleSize; }
+      inline void setMaxTimeSeconds(float seconds) { options_.max_time_seconds = seconds; }
+      inline void setOverlap(float overlap) { options_.configureOverlap(overlap); }
 
       /** \brief Get the fitness score of alignment. Range from 0-1, higher is better.
        */
       inline float getFitnessScore() const { return fitness_score_; }
+      inline float getDelta() const { return options_.delta; }
+      inline int getSampleSize() const { return options_.sample_size; }
+      inline float getMaxTimeSeconds() const { return options_.max_time_seconds; }
+      inline float getOverlap() const { return options_.getOverlapEstimation(); }
+      
+      inline OptionType getOptions() const { return options_; }
```

Comment:
You must return a reference here, otherwise the object will be copied and the variable `options_` will not be updated.
```suggestion
      inline OptionType& getOptions()  { return options_; }
      inline const OptionType& getOptions() const { return options_; }
```